docs/src/pages/docs/crocks/Either.md

```diff
@@ -996,7 +996,7 @@ Either c a ~> ((c -> b), (a -> b)) -> b
 
 Used as a means to map and extract a value from an `Either` based on the
 context, `either` takes two functions as its arguments. The first will map any
```

Comment:
Should `either` be capitalized?

Reply:
The first `Either` is capatilized because it's the type and the second `either` isn't because it's the function